src/snackbar.jsx

```diff
@@ -162,7 +161,7 @@ const Snackbar = React.createClass({
 
       //Only Bind clickaway after transition finishes
       this.timerTransitionId = setTimeout(() => {
-        this._bindClickAway();
+        this.timerTransitionId = 0;
```

Comment:
I understand. Let's leave like this. but add a comment to avoid future confusions. thanks :grin: